### PR TITLE
Issue 410

### LINKED ISSUE #410: modification of border block for https://sunstar.com/sustainability/social/supply-chain
Screenshot 2023-11-20 at 10 25 13 PM

### PR DESCRIPTION
**Pull request by hparwani2**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #410 

Test URLs:
- Before: https://main--sunstar--hlxsites.hlx.live/sustainability/social/supply-chain
- After: https://issue-410--sunstar--hlxsites.hlx.live/sustainability/social/supply-chain

- [ ]  If you are adding a new block or a variation to an existing block, has it been added in the [block-inventory](https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/Shared%20Documents/sites/sunstar/sunstar-engineering/_drafts/block-inventory.docx?d=w0e883b961c9f4401bae37da23ace83ec&csf=1&web=1&e=KcfAim) ?

### CODE REVIEW

File: blocks/text/text.css [hunk 1]
```diff
@@ -71,6 +71,31 @@ main .block.text.border {
   margin-bottom: 1rem;
 }
 
+main .block.text.rounded-border {
+  padding: 0 2rem;
+  margin-bottom: 1rem;
+}
+
+main .block.text.blue {
+  color: #007CAD;
```
Comment (sdmcraft) [inline blocks/text/text.css:80]: This has a token `secondary` in styles.css

File: blocks/text/text.css [hunk 2]
```diff
@@ -71,6 +71,31 @@ main .block.text.border {
...
+
+main .block.text.blue h4 {
+  color: #007CAD;
+}
+
+main .block.text.rounded-border p {
+  font-size: var(--body-font-size-xl);
+}
+
+main .block.text.block.text.rounded-border  h4 {
```
Comment (sdmcraft) [inline blocks/text/text.css:91]: why does `.block.text` repeat twice?

File: blocks/text/text.css [hunk 3]
```diff
@@ -71,6 +71,31 @@ main .block.text.border {
...
+  text-align: center;
+  border-bottom: 1px solid #3c4650;
```
Comment (sdmcraft) [inline blocks/text/text.css:93]: This has a token `light-black` in styles.css

File: blocks/text/text.css (hunk 1 above)
Comment (jindaliiita) [reply blocks/text/text.css:80]: Can we add this as a general variant on any block. WDYT ?

File: blocks/text/text.css (hunk 2 above)
Comment (jindaliiita) [reply blocks/text/text.css:92]: you can remove style
main .block.text.rounded-border

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**